com.valvesoftware.Steam.CompatibilityTool.Proton.metainfo.xml

```diff
@@ -5,9 +5,9 @@
   <name>Proton (community build)</name>
   <name xml:lang="pt_BR">Proton (construção da comunidade)</name>
   <name xml:lang="es">Proton (construcción comunitaria)</name>
-  <summary>Compatibility tool for Steam Play based on Wine and additional components</summary>
-  <summary xml:lang="pt_BR">Ferramenta de compatibilidade para Steam Play baseada no Wine e componentes adicionais</summary>
-  <summary xml:lang="es">Herramienta de compatibilidad para Steam Play basada en Wine y componentes adicionales</summary>
+  <summary>Latest upstream builds of Proton built on the Flatpak runtime</summary>
```

Comment:
@nanonyme What do you think? How we should name it so it would be clear what it is? "Latest Proton build on top of Freedesktop SDK" maybe?

Reply:
Can we have both freedesktop-sdk and flatpak mentioned or is it too long?

Reply:
"Latest Proton build on top of Freedesktop SDK flatpak runtime", then?